esp32/svd/patches/esp32.yaml

```diff
@@ -4,6 +4,8 @@ I2C0:
   _modify:
     INT_STATUS:
       name: INT_ST
+  _include:
+    - ../../../common_patches/i2c0.yaml
```

Comment:
This chip has multiple I2C controllers, I2C1 seems to be ignored entirely here (including the previous changes). We should probably fix this?

Reply:
Sorry, I may have missed something.
But from what I understand, in our `SVD`s (same in source files) `i2c1` is fully derived from `i2c0`, so every change for `i2c0` is "automatically" applied on `i2c1`
```
<peripheral derivedFrom="I2C0">
      <name>I2C1</name>
      <description>I2C (Inter-Integrated Circuit) Controller 1</description>
      <baseAddress>0x3FF67000</baseAddress>
      <interrupt>
        <name>I2C_EXT1</name>
        <value>50</value>
      </interrupt>
    </peripheral>
```